src/main/java/edu/greenblitz/robotName/subsystems/shooter/FlyWheel/simulationFlyWheel/SimulationFlyWheelConstants.java

```diff
@@ -5,20 +5,20 @@
 
 public class SimulationFlyWheelConstants extends FlyWheelConstants {
 
-    public static class RightMotor{
+    public static class RightMotor {
 
         public static final int NUMBER_OF_MOTORS = 1;
 
-        public static final double GEARING =  1.0 / 8;
+        public static final double GEARING = 1.0 / 8;
```

Comment:
in the real motors its the motor rotations / final rotations which means it should be 8

Reply:
oh its the simulation. in that case take the gear ratio value from the real fly wheel